subprojects/core-api/src/main/java/org/gradle/StartParameter.java

```diff
@@ -305,7 +305,7 @@ public void setBuildFile(@Nullable File buildFile) {
      * @return this
      */
     public StartParameter useEmptySettings() {
-        DeprecationLogger.nagUserOfDeprecated("StartParameter#useEmptySettings()");
+        DeprecationLogger.deprecate("StartParameter#useEmptySettings()").nagUser();
```

Comment:
These ones can probably be changed to `deprecateMethod`, however, this will change the deprecation message wording a bit and I am unsure if this is fine yet. It probably is, but I'd rather come back to this once I look deeper into particular deprecations for doc attachments.